package.json

```diff
@@ -8,9 +8,9 @@
     "@octokit/rest": "^15.8.2",
     "agenda": "^0.9.1",
     "github-url-to-object": "4.0.4",
-    "hubot": "^2.19.0",
-    "hubot-help": "^0.2.2",
-    "hubot-matteruser": "^4.4.0",
+    "hubot": "^3.1.1",
```

Comment:
dependabot suggested this upgrade in PR (#42)[https://github.com/gorkem/penfold/pull/42/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12]